doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+
+```"SiteName": "Mattermost"```
+*Name of service shown in login screens and UI.*
+
+```"MaxUsersPerTeam": 50```
+*Maximum number of users per team, including both active and inactive users.*
+
+```"EnableTeamCreation": true```
+*"true": Ability to create a new team is enabled for all users; “false”: the ability to create teams is disabled. The create team button displays an error when pressed.*
+
+```"EnableUserCreation": true```
+*"true": Ability to create new accounts is enabled via inviting new members or sharing the team invite link;“false”: the ability to create accounts is disabled. The create account button displays an error when trying to sign up via an email invite or team invite link.*
+
+```"RestrictCreationToDomains": ""```
+*Teams can only be created by a verified email from this list of comma-separated domains (e.g. "corp.mattermost.com, mattermost.org").*
+
+
+### SQL Settings
+___
+Settings to configure the data sources, connections, and encryption of SQL databases. Changing properties in this section will require a server restart before taking effect. 
+
+```"DriverName": "mysql"```
+*"mysql": enables driver to MySQL database; "postgres": enables driver to PostgreSQL database. This setting can only be changed from config.json file, it cannot be changed from the System Console user interface.*
+
+```"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8"```
+*This is the connection string to the master database. When DriverName set to "postgres" then use a connection string in the form “postgres://mmuser:password@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10”. This setting can only be changed from config.json file, it cannot be changed from the System Console user interface.*
+
+```"DataSourceReplicas": []```
+*This is a list of connection strings pointing to read replicas of MySQL or PostgreSQL database.  If running a single server, set to DataSource. This setting can only be changed from config.json file, it cannot be changed from the System Console user interface.*
+
+```"MaxIdleConns": 10```
+*Maximum number of idle connections held open to the database.*
+
+```"MaxOpenConns": 10```
+*Maximum number of open connections held open to the database.*
+
+```"Trace": false```
+*"true": Executing SQL statements are written to the log for development.*
+
+```"AtRestEncryptKey": "7rAh6iwQCkV4cA1Gsg3fgGOXJAQ43QVg"```
+*32-character (to be randomly generated via Admin Console) salt available to encrypt and decrypt sensitive fields in database.*
+
+
+### Email Settings
+___
+Settings to configure email signup, notifications, security, and SMTP options. 
+
+### Signup
+
+```"EnableSignUpWithEmail": true```
+*"true": Allow team creation and account signup using email and password; “false”: Email signup is disabled and users are not able to invite new members. This limits signup to single-sign-on services like OAuth or LDAP.*
```

Comment:
Some places we use "signup" others we use "sign up", maybe pick one and change them all to match?